pom.xml

```diff
@@ -1098,12 +1068,26 @@
                   <artifactId>maven-antrun-plugin</artifactId>
                   <version>3.1.0</version>
                   <executions>
+                      <execution>
+                          <id>get-buildvers</id>
+                          <goals>
+                              <goal>run</goal>
+                          </goals>
+                          <configuration>
+                              <target xmlns:ac="antlib:net.sf.antcontrib">
+                                  <property name="expression" value="${expression}"/>
```

Comment:
It's unclear to me how `expression` is being set.  It looks like this depends on the execution of the following `execution` block first.  If that's the case, it'd be great to order these execution blocks in the file to match the otder we would expect them to execute so it's a bit easier to follow.

Reply:
`expression` for this target will be set as a maven property when called from the CLI by `version-def.sh` or `buildall` script